### ISSUE TITLE
Document Kiali Istio registry requirements

### ISSUE THREAD

**Issue by nrfox** [kiali/kiali.io#567]
Kiali uses the Isito registry to gather information about the mesh. It'd be helpful to document exactly which registry endpoints are used by Kiali e.g. `/debug/registryz` and what, if any, are the auth requirements.

**Comment by jshaughn** [comment]
Good idea, it could possibly be a FAQ entry analogous to the required metrics FAQ.    Or maybe it should in in Installation?

**Comment by jshaughn** [comment]
BY the way, I'm not sure we want to use the kiali.io issues, maybe it should all be consolidated under kiali/kiali.

**Comment by jmazzitelli** [comment]
> BY the way, I'm not sure we want to use the kiali.io issues, maybe it should all be consolidated under kiali/kiali.

This is correct. All kiali issues go under kiali repo - included docs. Just use the "docs" label over there.
